tmt/steps/provision/podman.py

```diff
@@ -98,4 +98,5 @@ def prepare(self, how, what):
 
     def destroy(self):
         """ Remove the container """
-        self.podman(f'container rm -f {self.container_name}')
+        if self.container_name is not None:
```

Comment:
`if self.container_name` is sufficient :)

Reply:
Ok, let's make it short ;)